app/src/main/java/com/yterletskyi/happyfriend/features/settings/ui/SeparateLineItem.kt

```diff
@@ -0,0 +1,31 @@
+package com.yterletskyi.happyfriend.features.settings.ui
+
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Paint
+import androidx.core.view.get
+import androidx.recyclerview.widget.RecyclerView
+
+class SeparateLineItem : RecyclerView.ItemDecoration() {
+
+    private val paint = Paint().apply {
+        color = Color.parseColor("#d3d3d3")
+    }
+
+    override fun onDraw(c: Canvas, parent: RecyclerView, state: RecyclerView.State) {
+        val width = parent[parent.childCount - 1].width.toFloat()
+        val y = parent[parent.childCount - 1].y
+        c.drawLine(
+            0f + separator_line_width,
+            y - separator_line_height,
+            width - separator_line_width,
+            y - separator_line_height,
+            paint
+        )
+    }
+
+    companion object {
+        const val separator_line_height = 8f
+        const val separator_line_width = 45f
```

Comment:
also why is it 45? How did you calculate it?

Reply:
I subtracted from startX coordinate and endX coordinate this number in order to make  indent from start and end of screen. I didn't calculate this number, just made it on eye.

Reply:
ok got it. Making something on eye is not a best way in development, because our eye can be different from tester's eye or client's eye :) Let's try to calculate it.  For example:
we know the margin from start of the screen to the start of the text in xml (in dp). Let's try to subtract it.
<img width="425" alt="image" src="https://user-images.githubusercontent.com/11182351/187188819-61965c60-c549-4a01-a0c1-c1d36f775eaa.png">
What do you think about it?